### DISCUSSION TITLE
MSYS2 - installation problem

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2491] [Q&A Feedback]
Has anyone been able to install MSYS2 in Sandboxie Plus? Once the installer is launched it starts over and over again in infinite loop. I have internet access blocked and admin rights dropped for my sandbox (in case it has any impact in this case).

### Reopened as issue #2493

**Comment by offhub** [comment]
Try running `msys2-xxx.exe` via `sandboxed command prompt`.

1. Right click on the box > `Run > Command Prompt`
2. Type  `X:\example\path\msys2-xxx.exe` in the CMD window then press `ENTER`.
3. Close the prompt window.
4. Installation should start after you close the window.

**Reply by ghost** [reply]
Unfortunately it did not help. The same issue still occurs.
It looks like this: https://i.imgur.com/VLwrONX.png

**Reply by offhub** [reply]
Wait a moment before closing the command window. It may not work if you close the window immediately.

**Reply by ghost** [reply]
I did. The problem still occurs.

**Reply by isaak654** [reply]
Please open a new [problem report](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose) by filling in as many details as possible.
